README.md

```diff
@@ -153,25 +153,23 @@ If a Lord drops their hammer during the Flip phase they are instantly demoted to
+If there is at least one Lord and no active Uprising nails currently exist, any Lad may call "Uprising" at the start of their turn before flipping the hammer. If they do, they place one or more Uprising nails somewhere in the stump and hammer them to the same starting depth used for other nails. There should be one Uprising nail for every 7 players. I.e., the number of Uprising nails is equal to `Ceiling(n/7)` where `n` is the number of players (1-7 players = 1 nail, 8-14 players = 2 nails, etc).  Each Uprising nail must be placed at least 2 inches away from any other nails if possible. Otherwise, they should be placed in whatever area maximizes distance from nearby nails. Uprising rules apply for the remainder of that turn.
 
-As long as there is an active Uprising nail and the current player is a Lad, they may call "Uprising" before flipping the hammer. If they do, then the turn officially counts as an Uprising and Uprising rules apply for the remainder of the turn. 
+As long as there is at least one active Uprising nail and the current player is a Lad, they may call "Uprising" before flipping the hammer. If they do, then the turn officially counts as an Uprising, and Uprising rules apply for the remainder of the turn. 
 
-During an Uprising, the active player must perform a double hammer flip followed by one strike on the Uprising nail without hesitation (note that they do not get two strikes as they normally would). If this strike finishes the Uprising nail, the Uprising is considered complete and all Lords are immediately demoted to Lads and must begin new starting nails. If the active player has already finished their starting nail (i.e., the House of Lords was previously full), they instantly ascend to Lord. The Uprising nail is now no longer active and a new one may be started by future Lads.
+During an Uprising turn, the active player must perform one flip more than they normally would for a given number of strikes. In other words, the amount of strikes a player gets during an Uprising turn is `f - 1` where `f` is the number of flips performed (e.g., a double flip grants 1 strike). The first strike performed during an Uprising turn must be on an Uprising nail. Any other nails that are hit before a succesful strike on an Uprising nail are considered invalid and will incur penalties as outlined in the [Penalties](#ix-penalties) section. If the active player finishes the final Uprising nail, the Uprising is considered complete and all Lords are immediately demoted to Lads and must begin new starting nails. If that player has already finished their starting nail (i.e., the House of Lords was previously full), they instantly ascend to Lord. The Uprising is now no longer active and a new one may be started by future Lads.
```

Comment:
The syntax to link to another header requires replacing all spaces with dashes, so it has to be:

>[Penalties](#ix---penalties)

Reply:
>the Uprising is considered complete

also I vote for the clearly far superior term of "squanched" 😂